crates/turbopack-ecmascript/src/references/esm/dynamic.rs

```diff
@@ -71,10 +75,22 @@ impl ChunkableAssetReference for EsmAsyncAssetReference {
 
 #[turbo_tasks::value_impl]
 impl CodeGenerateable for EsmAsyncAssetReference {
+    #[turbo_tasks::function]
+    async fn code_generation(
+        &self,
+        _context: EcmascriptChunkingContextVc,
+    ) -> Result<CodeGenerationVc> {
+        bail!("Code generation for EsmAsyncAssetReference requires availability info")
+    }
+}
```

Comment:
Is it still necessary for this to implement `CodeGenerateable`?

Reply:
removed